syslibs/opcua/CMakeLists.txt

```diff
@@ -1,15 +1,22 @@
-
 add_ov_library(opcua)
 ov_library_includes(opcua ov ksbase)
 
+#alternative syntax:
+set(UA_NAMESPACE_ZERO "REDUCED" CACHE STRING "Enable reduced NS0 (default) to allow for faster server startup." FORCE)
+set(UA_ENABLE_AMALGAMATION "ON" CACHE BOOL "Merge als .c and .h files from open62541 in one open62541.c/.h" FORCE)
+set(UA_AMALGAMATION_ARCHITECTURES "posix;win32" CACHE STRING "Include posix and win32 architecture." FORCE)
```

Comment:
Why are the AMALGAMATION options required?

Reply:
I guess, its just for lazy including without the need for searching the right headers. I will change that.